### ISSUE TITLE
Weekly Inbrief Slide

### ISSUE THREAD

**Issue by kimisatoPC** [ICPI/Prevention-Cluster#27]
Dearest workstream leads, please submit any updates you would like included in the Prevention Cluster inbrief slide by Friday COB.

@aliyahwakil @shapaklyak @EReitenauer @erinschelar @kimisatoPC

**Comment by kimisatoPC** [comment]
### **Inbrief slide: Week of February 18**

Week of Feb 11 slide for reference:
![image](https://user-images.githubusercontent.com/36009724/52866867-98d5e000-310d-11e9-84d8-975cc14c488b.png)

**Comment by shapaklyak** [comment]
@kimisatoPC for OVC

- Working on [OVC Trends analysis](https://www.pepfar.net/OGAC-HQ/icpi/Shared%20Documents/Forms/AllItems.aspx?RootFolder=%2FOGAC%2DHQ%2Ficpi%2FShared%20Documents%2FClusters%2FPrevention%2FWorking%20Folder%2FOVC%2FOVC%20Trends&FolderCTID=0x01200080CC6F83D1766F4D9E3497D4529EC74A&View=%7B94C838B2%2DE166%2D4122%2DB8B4%2D7BEB9E1BC12B%7D), complications with how OVC SERV total numerator is stored for FY2017. Hidden axis for now since not QC'd
![image](https://user-images.githubusercontent.com/5234106/52867398-dab35600-310e-11e9-8082-3ae6ad07e137.png)

- [Updating changes](https://github.com/ICPI/Prevention-Cluster/blob/master/OVC/OVCTrends/OVC_SERVcalculations.md) on OVC SERV calculations for Results & Targets over the years

**Comment by aliyahwakil** [comment]
this is a helpful reminder. thanks! sending shortly

**Comment by kimisatoPC** [comment]
@shapaklyak @aliyahwakil @erinschelar let me know if you have anything to add!

![image](https://user-images.githubusercontent.com/36009724/53264612-081d7800-36aa-11e9-9f8e-6c7ce0779bcd.png)

**Comment by shapaklyak** [comment]
@kimisatoPC thank you, same as last week. no other updates for OVC at this time.

**Comment by aliyahwakil** [comment]
> @shapaklyak @aliyahwakil @erinschelar let me know if you have anything to add!
> 
> ![image](https://user-images.githubusercontent.com/36009724/53265066-3e0f2c00-36ab-11e9-84a1-a6d60013ad7c.png)

@kimisatoPC @erinschelar 1 edit: Updated VMMC dashboard with initial Q1 data from Genie and shared with QC'er. Will begin QC'ing when Panorama is released.

**Comment by kimisatoPC** [comment]
@shapaklyak @aliyahwakil Perfect- will update now. Thank you!

**Comment by kimisatoPC** [comment]
@shapaklyak @aliyahwakil @erinschelar anything to add?
![image](https://user-images.githubusercontent.com/36009724/54840848-4b1c3c80-4ca4-11e9-9491-45b827549e14.png)

**Comment by erinschelar** [comment]
Thanks! For OVC, can add that we are working with systems to clarify some discrepancies between DATIM and Panorama. 

Would it be worth adding discussion item on the question about Panorama Q2 changes that I emailed to Jasmine yesterday?

**Comment by shapaklyak** [comment]
DATIM issue in more detail: OVC SERV COP18/FY19 Targets for Top Level Numerator should be Active + Grad ONLY (by age/sex). All 4 disaggs should NOT be used to sum to the Top Level Numerator. The DATIM Target entry screen for FY18 and FY19 is taking the sum of all 4 program status disaggregates instead of just active and graduated. 

Feel free to reference issue #29 , #30 , #31

**Comment by kimisatoPC** [comment]
@erinschelar good idea to add pano q2 discussion item. @shapaklyak added in a special note to the slide

![image](https://user-images.githubusercontent.com/36009724/54849393-917c9600-4cba-11e9-9024-d405d899710d.png)

**Comment by aliyahwakil** [comment]
Prepared VMMC dashboard for clean Q1 update. 
Will update tool with Genie extract


Sorry if it's too late :/

**Comment by kimisatoPC** [comment]
@aliyahwakil not too late! Thank you!

**Comment by shapaklyak** [comment]
For Monday, 5/6
OVC 
•	Updating shell for OVC dashboard FY19 Q2
•	Pending MSD Pivots for OVC indicators
•	OVC_HIV STAT FY18 APR total numerator corrected in DATIM (APR = Q4) 

GBV QA
•	Virtually supporting GBV QA assessments in Uganda
